ansible/roles/schulcloud-server-core/tasks/main.yml

```diff
@@ -16,6 +16,12 @@
       namespace: "{{ NAMESPACE }}"
       template: api-files-svc.yml.j2
 
+  - name: EduSharingService
```

Comment:
Kein KamelCase verwenden im Namen, Leerzeichen und Bindestriche gehen

Reply:
Fixed by renaming `EduSharingDeployment`to `Edu Sharing Deployment`!